service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/BlobCopier.java

```diff
@@ -77,7 +75,7 @@ public BlobCopierResult copyBlobs(
         destStorageData.storageContainerResource().getResourceId(),
         destStorageData.storageContainerResource().getWorkspaceId());
     var pollResults =
-        blobPollers.map(blobPoller -> blobPoller.waitForCompletion(MAX_BLOB_COPY_POLL_TIMEOUT));
+        blobPollers.map(blobPoller -> blobPoller.waitForCompletion(SUBFLIGHT_TOTAL_DURATION));
```

Comment:
~Can you explain a bit more the motivation behind combining constants here? It seems like we'd want to allow these durations to vary independently.~ nm, I see that the AwaitCloneControlledAzureStorageContainerResourceFlightStep is polling on a subflight using this code, I think this makes sense.